yt/visualization/tests/test_geo_projections.py

```diff
@@ -0,0 +1,85 @@
+"""
+Tests for making unstructured mesh slices
+
+"""
+
+#-----------------------------------------------------------------------------
+# Copyright (c) 2013, yt Development Team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#-----------------------------------------------------------------------------
+from nose.plugins.attrib import attr
+
+import yt
+import unittest
+from yt.testing import ANSWER_TEST_TAG, fake_amr_ds, requires_module
+from yt.utilities.answer_testing.framework import GenericImageTest
+from yt.visualization.geo_plot_utils import transform_list, get_mpl_transform
+
+def setup():
+    """Test specific setup."""
+    from yt.config import ytcfg
+    ytcfg["yt", "__withintesting"] = "True"
+
+def compare(ds, field, idir, test_prefix, test_name, transform,
```

Comment:
This argument isn't used, is there a reason for it to be here?

Reply:
Woah, I totally forgot to carry that through. Thanks for pointing it out. I think it's resolved now!